tests/testthat/_snaps/learn.md

```diff
@@ -406,22 +440,35 @@
     Code
       dm::dm_get_all_fks(my_dm)
     Output
-      # A tibble: 3 x 5
-        child_table child_fk_cols parent_table  parent_key_cols  on_delete
-        <chr>       <keys>        <chr>         <keys>           <chr>    
-      1 oseba       id_nesreca    nesreca       id_nesreca       no_action
-      2 nesreca     upravna_enota upravna_enota id_upravna_enota no_action
-      3 oseba       upravna_enota upravna_enota id_upravna_enota no_action
+      # A tibble: 9 x 5
+        child_table child_fk_cols  parent_table  parent_key_cols  on_delete
+        <chr>       <keys>         <chr>         <keys>           <chr>    
+      1 Loan_Acc    account_id     acc           account_id       cascade  
+      2 Loan_Acc    loan_id        loan          loan_id          cascade  
+      3 Loan_Order  loan_id        loan          loan_id          cascade  
+      4 Loan_Trans  loan_id        loan          loan_id          cascade  
+      5 oseba       id_nesreca     nesreca       id_nesreca       cascade  
+      6 Loan_Order  order_id       order         order_id         cascade  
+      7 Loan_Trans  transaction_id trans         transaction_id   cascade  
+      8 nesreca     upravna_enota  upravna_enota id_upravna_enota cascade  
+      9 oseba       upravna_enota  upravna_enota id_upravna_enota cascade  
```

Comment:
similar change from `no_action` to `cascade` in here as well.

Reply:
I also checked the other catalogs/schemas. All good here as well.
![image](https://user-images.githubusercontent.com/19807339/236193238-6e1fda7d-8c78-4d92-a562-04b7de727bac.png)